modules/tide_user_expire/tide_user_expire.install

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @file
+ * Tide User Expire.
+ */
+
+/**
+ * Implements hook_uninstall().
+ */
+function tide_user_expire_uninstall() {
+  \Drupal::configFactory()->getEditable('tide_tide_user_expire.settings')->delete();
+}
```

Comment:
I think there is a extra tide in the naming above, shouldn't it be just `tide_user_expire.settings`

Reply:
Hi @MdNadimHossain 
yes, It's updated.